test/Common/TestHelpers.cs

```diff
@@ -31,6 +31,7 @@ internal static class TestHelpers
             Uri notificationUrl = null,
             HttpMessageHandler eventGridNotificationHandler = null)
         {
+            if (nameResolver == null) { nameResolver = new SimpleNameResolver(); }
```

Comment:
Why do we need to add the `SimpleNameResolver` to all end-to-end tests? We didn't require this before so I don't understand why your change would require it now.

Reply:
Thank you for improving the code in the other PR!